wordcount.c

```diff
@@ -0,0 +1,30 @@
+//Ashank Kumar, reads a file and prints the words in it
```

Comment:
Just to follow up, the reason for this is because you don't take all types of whitespace into consideration - like tabs and newlines!